hw2/03-form.js

```diff
@@ -1,3 +1,113 @@
+/**
+ * Naya Mairena
+ * Fall 2022
+ * CS465P Full Stack
+ */
 /** Exercise 03 - Form **/
 
-// Add your code here
+//Get the form tag from the html file.
+let form = document.querySelector('form');
```

Comment:
I love the challenge of doing this all in javascript.  Getting use to working with the dom can take some time but it seems like you've got the hang of it.